src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1763,6 +1763,20 @@ bool isContainingAssemblyInReferences(ISymbol s)
         /// within this compilation, the reference is considered to be used. Etc.
         /// The returned set is a subset of references returned by <see cref="References"/> API.
         /// The result is undefined if the compilation contains errors.
+        /// 
+        /// The effect of imported namespaces on result of this API depends on whether reporting of
+        /// unused imports is disabled for the compilation. The reporting of unused imports is disabled
+        /// if <see cref="ParseOptions.DocumentationMode"/> is set to <see cref="DocumentationMode.None"/>.
```

Comment:
nit: suggestion:
```suggestion
        /// if <see cref="ParseOptions.DocumentationMode"/> is set to <see cref="DocumentationMode.None"/> (i.e. when documentation is not processed).
```

Reply:
> nit: suggestion

I think ```DocumentationMode.None``` is well documented and I'd rather not duplicate its documentation here.